<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>392</width>
    <height>193</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QCheckBox" name="checkBox_Variables">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Copy Variables</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_BackgroundCalc">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Calculkate Background</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_Fit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Fit All Peaks</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_Save">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Save All Spectra</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_Done">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>100</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Done</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_Apply">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>40</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_SetPrefix">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>150</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>spectra_</string>
   </property>
  </widget>
  <widget class="QLabel" name="prefix_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Enter Prefix</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
